docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,329 @@ title: React Native SDK Migration Guide
+            key: 'value',
+        })
+
+    revenue(event);
+    ```
+You can also `setRevenue(6)` instead of `setPrice(3)` and `setQuantity(2)`.
+
+## Advanced topics
+
+### Identity
+
+#### Device ID
+
+As the maintenance React Native SDK is a wrapper of the maintenance iOS, maintenance Android SDK and maintenance Browser SDK and provides mappings from React Native to native SDK functions, device ID generation follows the native SDK of each platform. Learn more about device ID lifecycle of [maintenance iOS SDK](../../ios/#device-id-lifecycle) and [maintenance Android SDK](../../android/#device-id-lifecycle). You can also call `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()` to set ADID or App Set ID as device ID.
+
+The new React Native SDK initializes the device ID in the following order, with the device ID being set to the first valid value encountered:
+
+1. Device ID of in the configuration on initialization
+2. "deviceId" value from URL param, for example http://example.com/?deviceId=123456789. If it runs on Web.
+3. Device ID in cookie storage.
+4. A randomly generated 36-character UUID
+
+#### Advertising IDs
+
+Maintenance React Native SDK supports setting an advertising ID as device ID by `setAdvertisingIdForDeviceId()` or `setAppSetIdForDeviceId()`. The new React Native SDK tracks ADID  by default as `config.trackingOptions.adid` defaults to `true`. However, the new React Native SDK doesn't support App Set ID, IDFA, or IDFV.
+
+### COPPA
+
+You can enable COPPA control by `enableCoppaControl()` in maintenance React Native SDK. The new React Native SDK doesn't support that API but you can still enable COPPA:
+
+* For iOS, IDFA and IDFV aren't tracked. For Android, you can turn off ADID by setting `config.trackingOptions.adid` to `false`
+* You can use an [enrichment Plugin](../#enrichment-type-plugin) to delete city in the payload.
+* You can turn off IP address tracking by setting `config.trackingOptions.ipAddress` to `false`
+* Location (latitude and longitude) isn't tracked
+
+### Session management
+
+Amplitude [defines a session](https://help.amplitude.com/hc/en-us/articles/115002323627-Track-sessions-in-Amplitude#how-amplitude-defines-sessions) slightly differently between mobile and web applications.
+
+The maintenance React Native SDK follows mobile-style session tracking definition based on foreground and background switching. You can set session expiration time by `setMinTimeBetweenSessionsMillis()`. It also supports automatically log start and end session events by calling `trackingSessionEvents(true)`.
+
+The new React Native SDK follows web-style session tracking definition based on the last event. You can customize the timeout window by `config.sessionTimeout`. It doesn't support automatic start and end session event tracking, but session is still managed by the SDK. Events that are logged within the same session have the same session ID and Amplitude still groups events together by session.
```

Comment:
Sorry, I was wrong about this. https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/pull/819#discussion_r1239015014

Reply:
No worries. I was convicted and didn't know we have two types of session definitions until then. Reverted